crates/storage/src/lazy/mapping.rs

```diff
@@ -154,7 +166,8 @@ where
     where
         Q: scale::EncodeLike<K>,
     {
-        pull_packed_root_opt(&self.storage_key(&key))
+        ink_env::get_contract_storage::<(&Key, Q), V>(&(&KeyType::KEY, key))
+            .unwrap_or_else(|error| panic!("failed to get value in mapping: {:?}", error))
```

Comment:
```suggestion
            .unwrap_or_else(|error| panic!("Failed to get value in Mapping: {:?}", error))
```

I wonder if we can test this somehow. Maybe by clearing the underlying contract storage manually and then trying to pull. Really this shouldn't happen unless a storage migration messes up your storage layout